dio/test/interceptor_test.dart

```diff
@@ -302,12 +302,76 @@ void main() {
       expect(response.data['errCode'], 0);
     });
 
-    test('ImplyContentTypeInterceptor', () async {
-      final dio = Dio();
-      expect(dio.interceptors, isNotEmpty);
-      expect(dio.interceptors.first, isA<ImplyContentTypeInterceptor>());
-      dio.interceptors.removeImplyContentTypeInterceptor();
-      expect(dio.interceptors, isEmpty);
+    group('ImplyContentTypeInterceptor', () {
```

Comment:
```suggestion
    group(ImplyContentTypeInterceptor, () {
```
Group takes an object as description and calls toString on it. By just using the type directly the test will always have the correct name, and it will also get renamed it the class gets renamed